ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -310,14 +337,18 @@ runDataDiffusion tracers
                Left sd -> do
                    a <- Socket.getSocketName sd
                    case a of
-                        (Socket.SockAddrUnix path) ->
+                        (Socket.SockAddrUnix path) -> do
+                          traceWith dtDiffusionInitializationTracer $ CreatingSystemdSocketForUnixPath path
```

Comment:
`Snocket.localSnocket` is a pure operation which just wraps various socket / named pipes operation in a record, it cannot fail.  Still the message makes sense, but maybe it should be something like: `UsingSystemdSocket path`.